server/tests/cluster.rs

```diff
@@ -101,3 +103,35 @@ async fn cluster_name_mismatch() -> googletest::Result<()> {
 
     Ok(())
 }
+
+#[test(tokio::test)]
+async fn replicated_loglet() -> googletest::Result<()> {
+    let mut base_config = Configuration::default();
+    base_config.bifrost.default_provider = ProviderKind::Replicated;
+
+    let nodes = Node::new_test_nodes_with_metadata(
+        base_config.clone(),
+        BinarySource::CargoTest,
+        enum_set!(Role::Worker | Role::LogServer),
+        3,
+    );
+
+    let cluster = Cluster::builder()
+        .cluster_name("cluster-1")
+        .nodes(nodes)
+        .build()
+        .start()
+        .await?;
+
+    cluster.wait_healthy(Duration::from_secs(30)).await?;
+
+    for idx in 1..=3 {
+        assert!(cluster.nodes[idx]
+            .lines("PartitionProcessor starting up".parse()?)
```

Comment:
In a future pass, we could check that the log(s) all have an appropriate sequencer and nodeset in the metadata store. Maybe even register and invoke the mock service. I can look into that as I think there would be value in firing an end-to-end tracer request, however minimal, in this test.

Reply:
Sounds like a good idea :-)